src/test/directo.test.js

```diff
@@ -1,5 +1,84 @@
-const { retTrue } = require('../');
+const { Format, CommandResult } = require('../enums');
+const { Directo } = require('../directo');
 
-test('some placeholder test', () => {
-  expect(retTrue()).toBe(true);
+function badCommand(command) {
+  return () => {
+    expect(() => {
+      const d = new Directo();
+      d.addCommand(command);
+    }).toThrow();
+  }
+}
+
+function goodCommand(command) {
```

Comment:
Are `badCommand` and `goodCommand` a representation of invalid vs valid commands?

Reply:
These are really just quick-and-dirty wrappers to let me write less in tests. I don't quite see them as go-to examples.

Reply:
I wasn't thinking about this in the scope of tests. Name em whatever tickles your testing fancy.